src/control/server/mgmt_pool_test.go

```diff
@@ -393,6 +393,20 @@ func TestServer_MgmtSvc_PoolCreate(t *testing.T) {
 				TgtRanks:  []uint32{0, 1},
 			},
 		},
+		"successful creation with meta size": {
+			targetCount: 8,
+			req: &mgmtpb.PoolCreateReq{
+				Uuid:         test.MockUUID(1),
+				Tierbytes:    []uint64{100 * humanize.GiByte, 10 * humanize.TByte},
+				MetaBlobSize: 2 * humanize.GiByte,
+				Properties:   testPoolLabelProp(),
+			},
+			expResp: &mgmtpb.PoolCreateResp{
+				//TierBytes: []uint64{100 * humanize.GiByte, 10 * humanize.TByte},
+				//MetaBlobSize: 2 * humanize.GiByte,
+				//TgtRanks: []uint32{0, 1},
+			},
```

Comment:
A bit of a surprising result, if the response is empty.

Reply:
This is actually a strange feature of how the test works, I had left the contents of the response commented to remind myself to revisit it, then I subsequently forgot :-) thanks for reminding me.
On L536 the mock drpc client is setup to return tc.expResp, PoolCreate() then simply returns tc.expResp and then on L555 we basically compare tc.expResp with itself.

As it stands I think TestServer_MgmtSvc_PoolCreate doesn't cover much of mgmtSvc.poolCreate() - which has been updated over time to address a number of edge cases and now has multiple code paths not all of which are covered in the unit tests. TestServer_MgmtSvc_calculateCreateStorage, TestServer_MgmtSvc_PoolCreateAlreadyExists and TestServer_MgmtSvc_PoolCreateDownRanks do a good job of covering some of these paths but we should probably look at evaluating what is actually covered by the unit tests and addressing any gaps as a technical debt task: https://daosio.atlassian.net/browse/DAOS-14351.

As for the changes in the scope of this ticket, looks like the main purpose of TestServer_MgmtSvc_PoolCreate is to flush out errors in the request and only 4 of 16 test cases provide tc.expResp. I think it's best to remove tc.expResp and instead just focus on error checking, with 4 of 16 cases passing with no error.

Reply:
In the end I've just separated drpc return value from expected response to clarify intent at the cost of response value duplication in the 4 successful test cases.